src/operationsClientBrowser.ts

```diff
@@ -0,0 +1,437 @@
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import * as path from 'path';
+import * as gax from './gax';
+
+import {GaxCall} from './apitypes';
+import {createApiCall} from './createApiCall';
+import {PageDescriptor} from './descriptor';
+import {ProjectIdCallback} from 'google-auth-library/build/src/auth/googleauth';
+import {GoogleAuth} from 'google-auth-library';
+
+const configData = require('./operations_client_config');
+
+import {GrpcClient} from './browser';
+
+export const SERVICE_ADDRESS = 'longrunning.googleapis.com';
+const version = require('../../package.json').version;
+
+const DEFAULT_SERVICE_PORT = 443;
+const CODE_GEN_NAME_VERSION = 'gapic/0.7.1';
+const PAGE_DESCRIPTORS = {
+  listOperations: new PageDescriptor(
+    'pageToken',
+    'nextPageToken',
+    'operations'
+  ),
+};
+
+export interface OperationsClientOptions {
+  libName?: string;
+  libVersion?: string;
+  clientConfig: gax.ClientConfig;
+}
+
+export {
+  BundleDescriptor,
+  LongrunningDescriptor,
+  PageDescriptor,
+  StreamDescriptor,
+} from './descriptor';
+
+export {CallSettings, constructSettings, RetryOptions} from './gax';
+
+export {createApiCall} from './createApiCall';
+
+export interface ClientStubOptions {
+  servicePath: string;
+  port: number;
+  // TODO: use sslCreds?: grpc.ChannelCredentials;
+  // tslint:disable-next-line no-any
+  sslCreds?: any;
+}
+
+export class OperationsClient {
+  auth: GoogleAuth;
+  private _getOperation!: GaxCall;
+  private _listOperations!: GaxCall;
+  private _cancelOperation!: GaxCall;
+  private _deleteOperation!: GaxCall;
+
+  constructor(
+    gaxGrpc: GrpcClient,
+    // tslint:disable-next-line no-any
+    grpcClients: any,
+    options: OperationsClientOptions
+  ) {
+    const opts: OperationsClientOptions & ClientStubOptions = Object.assign(
+      {
+        servicePath: SERVICE_ADDRESS,
+        port: DEFAULT_SERVICE_PORT,
+        clientConfig: {},
+      },
+      options
+    );
+
+    const googleApiClient = ['gl-node/' + process.versions.node];
+    if (opts.libName && opts.libVersion) {
+      googleApiClient.push(opts.libName + '/' + opts.libVersion);
+    }
+    googleApiClient.push(
+      CODE_GEN_NAME_VERSION,
+      'gax/' + version,
+      'grpc/' + '0.4.3' // TODO: Add correct grpc client version dynamically into grpcClient
```

Comment:
Is this something we can do?

Reply:
Absolutely - we need to address it. Can we put `grpc-fallback/1.0.0` here? (@noahdietz any ideas welcome!)

Reply:
Ok sounds good